integration-tests/tests/circuits.rs

```diff
@@ -35,21 +52,143 @@ const CIRCUITS_PARAMS: CircuitsParams = CircuitsParams {
     keccak_padding: None,
 };
 
-async fn test_evm_circuit_block(block_num: u64) {
-    log::info!("test evm circuit, block number: {}", block_num);
+async fn gen_inputs(
+    block_num: u64,
+) -> (
+    CircuitInputBuilder,
+    eth_types::Block<eth_types::Transaction>,
+) {
     let cli = get_client();
     let cli = BuilderClient::new(cli, CIRCUITS_PARAMS).await.unwrap();
-    let (builder, _) = cli.gen_inputs(block_num).await.unwrap();
+
+    cli.gen_inputs(block_num).await.unwrap()
+}
+
+fn test_run<ConcreteCircuit: Circuit<Fr>, R: RngCore>(
```

Comment:
I did it like in here:

https://github.com/zcash/halo2/blob/main/halo2_proofs/src/plonk/prover.rs#L40

but I don't have a personal preference, I change it

Reply:
Ah, thanks for the pointer!  I would guess that they used that long name because `C` was already used?  Anyway, since you found this reference in the halo2 code (and the use case here is the same), I would leave it up to you to decide if you want to change it or not :)